dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -22,10 +22,21 @@ def new_general
       # Accept days 0 through 4. Day 5 is the post workshop survey and should use the new_post route
       day = params[:day].to_i
       workshop = get_workshop_for_new_general(params[:enrollmentCode], current_user)
+
       unless validate_new_general_parameters(workshop)
         return
       end
 
+      # Use Foorm for local summer workshops. This preserves the legacy url, which facilitators may
```

Comment:
I think we want this logic before the validation logic. Our new methods do their own validation and the new validation is slightly different (mainly our change to allow as many days as sessions).

Reply:
(if we want both urls to have the same behavior)

Reply:
Ah, I did this to make a test pass, because our check for `local_summer?` was failing on a `nil` `workshop`.

Reply:
But I guess I can change back and add a `nil` check.